macaroonbakery/tests/test_codec.py

```diff
@@ -8,13 +8,10 @@
 from nacl.encoding import Base64Encoder
 from nacl.public import PrivateKey
 
-from macaroonbakery import (
-    BAKERY_V1, BAKERY_V2, BAKERY_V3, codec, utils
-)
+import macaroonbakery
+from macaroonbakery import utils
```

Comment:
I'm not sure it's great that tests are importing a macaroonbakery internal python module.